Cargo.toml

```diff
@@ -53,6 +53,8 @@ members = [
 	"core/service/test",
 	"core/session",
 	"core/sr-api-macros",
+	"core/sr-arithmetic",
+	"core/sr-arithmetic/fuzzer",
```

Comment:
I am not sure with how this fuzzing stuff works, but can you double check if this is needed?

Reply:
Having this here just means that the fuzzer doesn't have it's own `Cargo.lock`. I'm happy to remove this.

Reply:
in parity-scale-codec the fuzzer is part of the workspace as well, I think this is fine